### PR TITLE
Restore BDFL Delegates

### PR DESCRIPTION
**Pull request by bitwiseman**

### CODE REVIEW

**Comment by bitwiseman** [comment]
NOTE: I've opened an addendum to this PR. https://github.com/jenkinsci/bitwiseman/pull/1 attempting to address feedback from @oleg-nenashev .

**Comment by omehegan** [comment]
@bitwiseman bad link?

**Comment by bitwiseman** [comment]
@omehegan  https://github.com/bitwiseman/jep/pull/1 Thanks!

File: jep/1/README.adoc [hunk 1]
```diff
@@ -185,25 +184,25 @@ For simplicity, this document uses the singular
...
+<<header-delegate, "BDFL-Delegate" header>> in the JEP.
 
-If a contributor other than the BDFL will be the Reviewer for the JEP,
-they should be selected before the JEP is submitted.
+A JEP's <<Sponsor>> cannot also be the BDFL Delegate for that JEP.
 
-If the Reviewer for a JEP is a contributor other than the BDFL,
-this will be recorded in the <<header-reviewer, "Reviewer" header>> in the JEP.
+In all cases where this document refers to the "BDLF" making a decision regarding a JEP,
+it means "the BDFL or BDFL Delegate for this JEP."
```
Comment (oleg-nenashev) [inline jep/1/README.adoc]: Another entity like "Reviewer" would help, but OK.

File: jep/1/README.adoc [hunk 2]
```diff
@@ -185,25 +184,25 @@ For simplicity, this document uses the singular
...
 then that contributor will have the authority to accept (or reject) that JEP.
-The BDFL may also choose to be the Reviewer for a JEP, overriding any self-nominations.
-If no other contributor volunteers, or if the core contributors (including the BDFL)
-cannot agree on a suitable Reviwer, the role reverts to the BDFL.
+The BDFL Delegate for a JEP will be recorded in the
+<<header-delegate, "BDFL-Delegate" header>> in the JEP.
 
-If a contributor other than the BDFL will be the Reviewer for the JEP,
-they should be selected before the JEP is submitted.
+A JEP's <<Sponsor>> cannot also be the BDFL Delegate for that JEP.
```
Comment (kohsuke) [inline jep/1/README.adoc]: Does this mean if I ask Oleg to be the Delegate for the remoting space in general, Oleg cannot sponsor a JEP to make a change in remoting?

Comment (kohsuke) [reply jep/1/README.adoc]: FWIW, I want Oleg to be able to sponsor a change in remoting, drive toward consensus, and accept that JEP himself.

Comment (bitwiseman) [reply jep/1/README.adoc]: @kohsuke - fair point.

**Comment by bitwiseman** [comment]
@oleg-nenashev Re Dispute resolution: yes, it makes sense to be more explicit.

File: jep/1/README.adoc [hunk 3]
```diff
@@ -185,25 +184,26 @@ For simplicity, this document uses the singular
...
 then that contributor will have the authority to accept (or reject) that JEP.
-The BDFL may also choose to be the Reviewer for a JEP, overriding any self-nominations.
-If no other contributor volunteers, or if the core contributors (including the BDFL)
-cannot agree on a suitable Reviwer, the role reverts to the BDFL.
+The BDFL Delegate for a JEP will be recorded in the
+<<header-delegate, "BDFL-Delegate" header>> in the JEP.
 
-If a contributor other than the BDFL will be the Reviewer for the JEP,
-they should be selected before the JEP is submitted.
+A JEP's <<Sponsor>> may also be the BDFL Delegate for that JEP,
```
Comment (oleg-nenashev) [inline jep/1/README.adoc]: LGTM. Now we have an escalation process described, so it should be fine

**Pull request merged**